files/slave.dockerfile

```diff
@@ -25,13 +24,24 @@ RUN yum install -y python-pip python-devel && \
     yum groupinstall -y 'Development Tools' && \
     # Very specific version is required for older Openstack Env.
     pip install --no-cache-dir --disable-pip-version-check shade==1.12.1 && \
+
+    # Use Ansible from source until rpm available w/ fix for:
+    # https://github.com/ansible/ansible/issues/15915
+    pip install --upgrade setuptools && \
+    git clone https://github.com/ansible/ansible.git \
+        --branch=stable-2.2 --recursive --depth=0 --single-branch \
+        /usr/src/ansible && \
+    cd /usr/src/ansible && \
+    make install && \
+    
     # Keep every layer as clean (i.e. small) as possible
-    yum history -y undo $(cat /root/trans_num) && \
+    yum history -y undo $(cat /root/trans_num) ; \
```

Comment:
This keeps the kommandir-container (aka slave) image small.  There's no need to hang onto all the development bits needed to build ``andible`` and ``shade``.  Containers based on this image are spun up and removed frequently, so keeping the image small improves performance substantially.